gensrc/proto/lake_types.proto

```diff
@@ -31,23 +23,20 @@ message TabletMetadataPB {
     optional int64 flags = 3; 
     optional TabletSchemaPB schema = 4;
     repeated RowsetMetadataPB rowsets = 5;
-    optional PrimaryIndexMetadataPB primary_index = 6;
-    optional uint32 next_rowset_id = 7;
+    optional uint32 next_rowset_id = 6;
```

Comment:
Can't change their ordinal because of compatibility

Reply:
Lake table not released yet~